app/frontend/components/_buttons.scss

```diff
@@ -0,0 +1,7 @@
+.button {
```

Comment:
I'd rather keep all the overrides in one place. We already have `app/frontend/foundation/overrides/buttons.scss`. Can you see if you can merge the two of these together?